tests.py

```diff
@@ -89,9 +89,6 @@ def test_plone_addon_template(self):
                 self.project + '/src/collective/foo/profiles/default/browserlayer.xml',  # noqa
                 self.project + '/src/collective/foo/profiles/default/collectivefoo_default.txt',  # noqa
                 self.project + '/src/collective/foo/profiles/default/metadata.xml',  # noqa
-                self.project + '/src/collective/foo/profiles/uninstall',  # noqa
```

Comment:
`profiles/uninstall` is only created for Plone 5 packages. Or rather, when `plone.is_plone5` is False, we delete it afterwards. Until now our answer to that question in the tests was always wrongly interpreted to mean True, so it got generated.

Probably the uninstall profile should simply always be generated. IIRC, on Plone 4 we would need an extra uninstall method in `Extensions`, and on Plone 5 that is not needed.